packages/ai-workspace-common/src/components/canvas/launchpad/config-manager/index.tsx

```diff
@@ -104,6 +104,8 @@ const ConfigItem = React.memo(
           placeholder={placeholder}
           value={initialValue !== null ? String(initialValue) : undefined}
           className="bg-transparent hover:bg-transparent focus:bg-transparent"
+          // @ts-ignore - 支持密码类型输入
```

Comment:
![image](https://github.com/user-attachments/assets/2cfd19e5-a01e-4cb4-848e-cddf3d69a5e4)
You can declare it in inputProps of schema.ts, no need to use ts-ignore here

operate path:
1. open `packages/openapi-schema/schema.yml`
2. find inputProps
3. add extra `password` types
4. at refly project root path run: `pnpm codegen` to generate types
5. remove `@ts-ignore`